inst/Upgrades.qml

```diff
@@ -13,7 +13,6 @@ Upgrades
 				ChangeRename { from:	"groupingVariable";									to:		"group"									}
 				ChangeRename { from:	"students";											to:		"student"								}
 				ChangeRename { from:	"welchs";											to:		"welch"									}
-				ChangeRename { from:	"mannWhitneyU";										to:		"wilcoxon"								}
```

Comment:
you need to add a new upgrade from `0.17.0` to `0.17.1` where you rename `mannWhitneyU` to `wilcoxon`.